src/coreclr/jit/gentree.cpp

```diff
@@ -21054,6 +21150,225 @@ GenTree* Compiler::gtNewSimdCreateBroadcastNode(
     return gtNewSimdHWIntrinsicNode(type, op1, hwIntrinsicID, simdBaseJitType, simdSize, isSimdAsHWIntrinsic);
 }
 
+GenTree* Compiler::gtNewSimdCreateScalarNode(
```

Comment:
Needs a header comment.

(I realize some (most) other function here don't have them, but we should still add them for new code)

Reply:
Fixed.